### PR TITLE
Created v6 sample

### PR DESCRIPTION
**Pull request by petersgiles**
@SimonCropp I've created a v6 host sample please review

### CODE REVIEW

File: samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
```
Comment (SimonCropp) [inline samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs]: use the code based api instead http://docs.particular.net/nservicebus/errors/#configure-your-error-queue-using-code

File: samples/hosting/nservicebus-host/Version_6/Sample/app.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="NServiceBus.Core" publicKeyToken="9fc386479f8a226c" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
```
Comment (SimonCropp) [inline samples/hosting/nservicebus-host/Version_6/Sample/app.config]: why the redirect?

Comment (petersgiles) [inline samples/hosting/nservicebus-host/Version_6/Sample/app.config]: It must have added it during nuget-if-ication. you want me to cut it?

Comment (SimonCropp) [inline samples/hosting/nservicebus-host/Version_6/Sample/app.config]: Yeah delete the file

**Comment by petersgiles** [comment]
I'm re updating the core in the host and will revisit this again when im done

**Comment by petersgiles** [comment]
@SimonCropp ready for merge

File: samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs (hunk 1 above)
Comment (SimonCropp) [inline samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs]: @petersgiles ping

Comment (petersgiles) [inline samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs]: reading it

File: samples/hosting/nservicebus-host/Version_6/Sample/MessageSender.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using NServiceBus;
+
+public class MessageSender : IWantToRunWhenBusStartsAndStops
+{
+    public async Task Start(IBusSession session)
+    {
+       await session.SendLocal(new MyMessage());
+    }
+
+    public  Task Stop(IBusSession session)
```
Comment (SimonCropp) [inline samples/hosting/nservicebus-host/Version_6/Sample/MessageSender.cs]: Kill the extra space

File: samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs (hunk 1 above)
Comment (SimonCropp) [inline samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs]: Yep

**Pull request merged**